common-requests/report-an-issue.md

```diff
@@ -1,8 +1,16 @@
 ---
 layout: page
 title: "Report an Issue"
-description: "TBD"
+description: ""
+tags:
+ - Help
+ - Issue
+ - Report
+author: clementkng
 ---
 
-# Report an Issue
+## Report an Issue
 
+If you would like to report an issue, visit the Github [issues section](https://github.com/open-horizon/open-horizon.github.io/issues), click the New issue button, and then fill out issue details.
+
+Please fill out a short, descriptive title for the issue and use the bigger text box to describe details such as current behavior, desired behavior, and what browser version you're seeing the issue on.
```

Comment:
`what browser version you're seeing the issue on`

this line presumes the environment is a web browser.  Maybe instead we generalize a bit and instead say something like:

`operating system and version, architecture (arm32, arm64, x86, power, z1, quantum), device, and any other operating parameters that will help us replicate the issue`

Reply:
Thanks for the feedback, done.